File: 2745-construct-the-longest-new-string/2745-construct-the-longest-new-string.scala
object Solution {
    def longestString(x: Int, y: Int, z: Int): Int = {
        var ans = 0
        if (x == y)
            ans = 2 * math.min(x, y)
        else
            ans = 2 * math.min(x, y) + 1

        ans += z

        ans * 2
    }
}

File: 2745-construct-the-longest-new-string/2745-construct-the-longest-new-string.cs
public class Solution {
    public int LongestString(int x, int y, int z) {
        int ans = 0;
        if (x == y)
            ans = 2 * Math.Min(x,y);
        else
            ans = 2 * Math.Min(x,y) + 1;
            
        ans += z;

        return ans * 2;        
    }
}